spec/shared/json_convertible_spec.rb

```diff
@@ -101,22 +101,40 @@ module FastlaneCI
     end
 
     it "Allows to object dictionary with custom property mapping" do
-      allow(MockJSONConvertible).to receive(:attribute_key_name_map).and_return({ :@one_attribute => "json_one_attribute" })
+      allow(
+        MockJSONConvertible
+      ).to receive(:attribute_key_name_map).and_return({ :@one_attribute => "json_one_attribute" })
       json_from_mock_object = mock_object.to_object_dictionary
       expect(json_from_mock_object).to eql({ "json_one_attribute" => "Hello", "other_attribute" => Time.at(0) })
     end
 
     it "Allows to perform custom mappings of the values for a given attribute when converting to JSON" do
-      allow(MockJSONConvertible).to receive(:attribute_name_to_json_proc_map).and_return({ :@other_attribute => proc { |timestamp| timestamp.strftime("%G-W%V-%uT%R%:z") } })
+      allow(
+        MockJSONConvertible
+      ).to receive(:attribute_name_to_json_proc_map).and_return(
+        { :@other_attribute => proc { |timestamp| timestamp.strftime("%G-W%V-%uT%R%:z") } }
+      )
       json_from_mock_object = mock_object.to_object_dictionary
-      expect(json_from_mock_object).to eql({ "one_attribute" => "Hello", "other_attribute" => Time.at(0).strftime("%G-W%V-%uT%R%:z") })
+      expect(
+        json_from_mock_object
+      ).to eql({ "one_attribute" => "Hello", "other_attribute" => Time.at(0).strftime("%G-W%V-%uT%R%:z") })
     end
 
     it "Allows to perform custom property mapping and value mapping at the same time" do
-      allow(MockJSONConvertible).to receive(:attribute_key_name_map).and_return({ :@one_attribute => "json_one_attribute", :@other_attribute => "json_other_attribute" })
```

Comment:
I think these are fine

Reply:
Alright, we could exclude specs from the 120 limit. Formatting rspec is weird with the `to ` syntax because of the space 😳